homeassistant/components/ialarmxr/translations/en.json

```diff
@@ -0,0 +1,23 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "ialarmxr_no_user_input_data_form": "No user input data form",
+            "cannot_connect": "Failed to connect",
+            "ialarmxr_generic_exception": "iAlarmXR generic error, please read the logs",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
```

Comment:
Please regenerate this file https://developers.home-assistant.io/blog/2020/04/10/translation-handling/?_highlight=script.translations%20develop